src/validation.cpp

```diff
@@ -1326,6 +1325,23 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::
     return PackageMempoolAcceptResult(package_state, std::move(results));
 }
 
+PackageMempoolAcceptResult MemPoolAccept::AcceptSubPackage(const std::vector<CTransactionRef>& subpackage, ATMPArgs& args)
+{
+    ATMPArgs single_args = ATMPArgs::SingleInPackageAccept(args);
+    AssertLockHeld(::cs_main);
+    AssertLockHeld(m_pool.cs);
+    if (subpackage.size() > 1) {
```

Comment:
I think this check doesn’t have test coverage as tx are individually validated in `AcceptPackage`.

Reply:
I think there is confusion, my comment is about the test coverage of the exact code path in the first half of `AcceptSubPackage` not of the second usage of `AccetSubPackage()`.

From a quick look on the tests at c2c9dfe9, it was uncertain.